File: applications/frontend/src/calculator.ts
/** Add two numbers. */
export function add(a: number, b: number): number {
  return a + b;
}

export function divide(a: number, b: number): number {
  return a / b;
}

export function isZero(value: number): boolean {
  var result = value == 0;
  return result;
}

export function unusedHelper(items: (number | null)[]): number[] {
  const result: number[] = [];
  for (const item of items) {
    if (item != null) {
      result.push(item);
    }
  }
  return result;
}

File: applications/backend/src/calculator.py
def add(a: int, b: int) -> int:
    return a + b


def divide(a: int, b: int) -> float:
    return a / b


def unused_helper(items: list) -> list:
    result = []
    for item in items:
        if item != None:
            result.append(item)
    return result
